pix-editor/app/models/challenge.js

```diff
@@ -252,10 +256,10 @@ export default class ChallengeModel extends Model {
     }
     const data = this._getJSON(ignoredFields);
     data.author = [this.config.author];
-
     data.status = 'proposé';
-    data.skill = this.skill;
+    data.skill = await this.skill;
```

Comment:
Ici on ne chargeait pas la donnée avant (pas de `await`), on pourrait donc peut-être utiliser un `this.belongsTo('skill').value()` à la place.